docs/azure-saas-docs/content/en/resources/documentation-build-process.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Let's get rid of this page and move the content into a readme under the docs folder.